### PR TITLE
Introduce prettier as pre-commit

### PR DESCRIPTION
**Pull request by Markionium**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`
- [x] Microsoft Alias (if you have one): mapol

#### Description of changes

Prettier (will update description :D)

### CODE REVIEW

**Comment by kenotron** [comment]
This is awesome - I'm looking for a [prettierrc / editorconfig](https://prettier.io/docs/en/configuration.html) file that will lock the team to some convention? Unless we've already all agreed on a convention?

**Comment by Markionium** [comment]
@kenotron It's currently here, but this is just based on the defaults and some small adjustments to keep them somewhat closer to the original fabric style. Feel free to suggest other things you think it should contain ;)

 https://github.com/Markionium/office-ui-fabric-react/blob/1ac7cf81de07917d4299f5fbcda302ed430a03f2/packages/office-ui-fabric-react-tslint/prettier.config.js

**Comment by kenotron** [comment]
LGTM!

**Comment by mtennoe** [comment]
This is awesome Mark!

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ script:
   - if [ $TRAVIS_BRANCH == "master" ] && [ $TRAVIS_PULL_REQUEST != "false" ];
     then npm run checkchange;
     fi
-  - npm run buildfast
+  - npm run buildfast -- -- --lint
```
Comment (cliffkoh) [inline .travis.yml]: hmm why so many `--`? Would two pairs be enough? i.e. `-- --lint`

**Pull request merged**